src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -6,13 +6,75 @@ import (
 	"io/ioutil"
 	"net"
 	"os"
+	"os/exec"
 	"path/filepath"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/moby/hyperkit/go"
 	"github.com/satori/go.uuid"
 )
 
+var defaultVPNKits = []string{"vpnkit",
+	"/usr/local/bin/vpnkit",
+	"/Applications/Docker.app/Contents/Resources/bin/hyperkit"}
+
+func createListenSocket(path string) (*os.File, error) {
```

Comment:
could we move the vpnkit functions further to the end. I seem to recall that in go you are supposed to have the exported/main functions first and have the internal ones later in the file. To me as a C programmer that's weird, but hey...I certainly wouldn't put them at the start as the main purpose of this file to run hyperkit